### PR TITLE
daemon: fix ${CLI_OPTS} -> ${CLI_OPTS[@]}

### PR DESCRIPTION
**Pull request by AlessandroLorenzi**

### CODE REVIEW

File: ceph-releases/luminous/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ start_osd() {
   fi
 
   if [ "$mode" = "forego" ]; then
-   echo "${CLUSTER}-${OSD_ID}: /usr/bin/ceph-osd ${CLI_OPTS} -f -i ${OSD_ID} --setuser ceph --setgroup disk" | tee -a /etc/forego/"${CLUSTER}"/Procfile
+   echo "${CLUSTER}-${OSD_ID}: /usr/bin/ceph-osd ${CLI_OPTS[@]} -f -i ${OSD_ID} --setuser ceph --setgroup disk" | tee -a /etc/forego/"${CLUSTER}"/Procfile
```
Comment (leseb) [inline ceph-releases/luminous/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh]: Use `[*]` instead.

Comment (AlessandroLorenzi) [inline ceph-releases/luminous/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh]: Only in this line or in all lines?

Comment (leseb) [inline ceph-releases/luminous/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh]: Only this one.

**Comment by AlessandroLorenzi** [comment]
thank you :)

**Pull request merged**